airflow/dags/download_gtfs_schedule_v2/download_schedule_feeds.py

```diff
@@ -17,23 +18,24 @@
 from calitp.config import is_development
 from calitp.storage import (
     get_fs,
-    AirtableGTFSDataExtract,
-    AirtableGTFSDataRecord,
     GTFSScheduleFeedExtract,
     GTFSFeedType,
     download_feed,
     ProcessingOutcome,
     SCHEDULE_RAW_BUCKET,
     PartitionedGCSArtifact,
     JSONL_EXTENSION,
+    GTFSDownloadConfig,
+    get_latest,
+    GTFSDownloadConfigExtract,
 )
 from pydantic import validator
 
 GTFS_FEED_LIST_ERROR_THRESHOLD = 0.95
 
 
-class AirtableGTFSDataRecordProcessingOutcome(ProcessingOutcome):
-    airtable_record: AirtableGTFSDataRecord
+class GTFSDownloadOutcome(ProcessingOutcome):
+    config: GTFSDownloadConfig
```

Comment:
I think this is what confuses me... If we are assuming that the file will have a valid config metadata, how is that different than actually backfilling valid config files? Like, when we backfill the actual schedule data (from the old downloader), how will we construct this config?

Reply:
We discussed this today; the tl;dr is that the config _extracts_ are a different thing than config _records_ as metadata on downloaded files. We plan to "backfill" the metadata but not the extracts themselves.